spinnakerdatadog/event_handlers.go

```diff
@@ -3,6 +3,7 @@ package spinnakerdatadog
 import (
 	"bytes"
 	"fmt"
+	"strings"
 
 	"github.com/bobbytables/spinnaker-datadog-bridge/spinnaker"
 	"github.com/bobbytables/spinnaker-datadog-bridge/spinnaker/types"
```

Comment:
should the imports that point to the upstream be changes to point at this fork?

Reply:
probably. I'm wondering if upstream will find this useful though, and the PRs share a branch at the moment, so it would break the PR to the upstream repo.